ikelani421/SmartVisitorManagementFend
# Mockup for Verify Guest Page

#### What does this PR do?
> - Build Mockup for Verify Guest Page
#### Description of Task to be completed?
> - Build Mockup for Verify Guest Page
> - Build Mockup to Collect Guest information
#### How should this be manually tested?
> - Clone the repo using the command `git clone [repo url]`,
> - run `npm install` to install the required dependencies
> - run `npm start` to start up the development server
> - navigate to `http://localhost:4000` on your browser to view the app
> - navigate to `http://localhost:4000/verify-guest` to view the verify guest page
#### Any background context you want to provide?
> - None
#### What are the relevant pivotal tracker stories?
> - None
#### Screenshots (if appropriate)
<img width="1680" alt="Screenshot 2020-04-16 at 14 44 52" src="https://user-images.githubusercontent.com/58782555/79463920-5b126700-7ff1-11ea-9170-5487c267fff2.png">
<img width="576" alt="Screenshot 2020-04-16 at 14 45 22" src="https://user-images.githubusercontent.com/58782555/79475884-2fe34400-8000-11ea-9c19-6aaaee3a4d33.png">

<img width="376" alt="Screenshot 2020-04-16 at 14 47 15" src="https://user-images.githubusercontent.com/58782555/79475900-383b7f00-8000-11ea-8455-b9591c012731.png">
<img width="1680" alt="Screenshot 2020-04-16 at 14 47 29" src="https://user-images.githubusercontent.com/58782555/79475905-3a9dd900-8000-11ea-9397-a2074fb97517.png">
<img width="579" alt="Screenshot 2020-04-16 at 14 48 05" src="https://user-images.githubusercontent.com/58782555/79475922-3ffb2380-8000-11ea-8b9a-46cace125d10.png">

#### Questions:
